### PR TITLE
Remove api calls from self.init

### PR DESCRIPTION
**Pull request by gsvr**
- [ ] self.districts
- [ ] self.array_emis

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: go-app.js [hunk 1]
```diff
@@ -16,20 +16,32 @@ go.rht = function() {
...
+                            if (result === true) {
+                                return "reg_emis_validates";
+                            } else if (opts.retry === false) {
+                                return "reg_emis_retry_exit";
+                            } else if (opts.retry === true) {
+                                return "reg_exit_emis";
+                            }
+                        });
+
+                    // if (go.utils.check_valid_emis(content, im)) {
```
Comment (imsickofmaps) [inline go-app.js]: rerun grunt compile?

**Comment by imsickofmaps** [comment]
fix comment the :+1: and land please

**Comment by gsvr** [comment]
@imsickofmaps Please review from 'fix district load'.  Last two were removing comments.

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**